stable/filebeat/values.yaml

```diff
@@ -145,6 +145,17 @@ podSecurityPolicy:
 ## Dont forget to enable http on config.http.enabled (exposing filebeat stats)
 monitoring:
   enabled: true
+  serviceMonitor:
+    # When set true and if Prometheus Operator is installed then use a ServiceMonitor to configure scraping
+    enabled: true
+    # Set the namespace the ServiceMonitor should be deployed
+    namespace: monitoring
+    # Set how frequently Prometheus should scrape
+    interval: 30s
```

Comment:
It's a better idea to have this not set and inherit the value from the default prometheus configuration. Otherwise having metrics come in at different intervals can be surprising.

Reply:
Thank you I will commit an improvement for this when I'm back at my dev machine later today 👍

Reply:
Pushed a new commit to remove `interval` 👍

Reply:
Not suggesting you remove these entirely, just make it not overridden by default. I don't have rights on this chart, so just making suggestions

Reply:
@thomasriley do you want to add those back in or are you good with it  as it i?  If you are good with it I'll give it a lgtm.

Reply:
On reflection I still agree with what @vsliouniaev originally suggested and I'd not actually override them in our environment however I have added them both back in as optional values for what its worth. Thanks! 👍